### ISSUE TITLE
Questions for label_base.json

### ISSUE THREAD

**Issue by greenyjwk** [microsoft/BiomedParse#76]
First of all, thank you for your great work.

I have a question regarding with label_base.json file.

I am trying to fine-tune with my own dataset(microbleeds Brain MRI images).
And for the label_base.json I made is like below. 
```
{
  "0": {
    "name": "background"
  },
  "1": {
          "name": "cmb",
          "parent": "abnormality",
          "alternative name": [
                "brain microbleeds",
                "cerebral microbleeds"
        ]
  }
}
```

But I don't know if I made this JSON file correctly. **Especially, I doubt if the "name", "parent" or "child" should be from the target_dist.json file.**

or 

should I make name:"cmb" be one of BIOMED_CLASSES in constant.py script, so it would be like below? But in that case, I am not sure what I would have to have as a parent and alternative name.
```
{
  "0": {
    "name": "background"
  },
  "1": {
          "name": "other abnormality",
          "parent": "abnormality",
          "alternative name": [
                "brain microbleeds",
                "cerebral microbleeds"
        ]
  }
}
```

For your reference, the image file format I created is 
[filename]_MRI_Brain.png

And the mask file format is
[filename]_MRI_Brain_cmb.png
 

Would you explain how I am supposed to make label_base.json file?

**Comment by theodore-zhao** [comment]
Hi,

You don't need to make additional label_base.json for finetuning. All you need is image and text prompt pairs, as illustrated in DATASET.md.

**Comment by greenyjwk** [comment]
Thank you for the response.

Okay, then I will assume that I don't have to modify label_base.json. 

And where am I supposed to specify the prompt I want? I have assumed that it is specified in the train.json and test.json. Because I can see the two keys "sent" and "raw". would you correct it please?

**Comment by theodore-zhao** [comment]
"sent" is good enough, and the two keys are essentially the same.

**Comment by greenyjwk** [comment]
Let me ask one more quick question about label_base.json. 

When I tried fine-tuning without modifying label_base.json, I got the error that the target I tried to fine-tune(Let's say it is 'Microbleeds') is not found.

So I appended the target which I want to fine-tune into the at the end of the label_base.json, like below. Wouldyou think it is correct way that I did?

```  
...
...
...
  "16": {
    "name": "other",
    "child": [
      "surgical tool",
      "public symphysis",
      "fetal head"
    ]
  },

  "17": {
    "name": "cmb",
    "parent": "abnormality",
    "alternative name": [
      "brain microbleeds",
      "cerebral microbleeds"
    ]
  }
```

**Comment by theodore-zhao** [comment]
Maybe you can just put "cerebral microbleeds" under "other abnormality".
